cmd/virt-launcher/virt-launcher.go

```diff
@@ -190,6 +191,11 @@ func initializeDirs(virtShareDir string,
 		panic(err)
 	}
 
+	err = ignition.SetLocalDirectory(ephemeralDiskDir + "/cloud-init-data")
```

Comment:
ignition needs a different directory from cloud-init.

Reply:
i fixed the directory. where can i check for those timeout errors. think i fixed it but dont know where to check...